plugins/discovery-ec2/src/main/java/org/opensearch/discovery/ec2/AwsEc2SeedHostsProvider.java

```diff
@@ -119,41 +126,47 @@ public List<TransportAddress> getSeedAddresses(HostsResolver hostsResolver) {
     }
 
     protected List<TransportAddress> fetchDynamicNodes() {
+        logger.info(
+            "fetching nodes from IMDS (instance-states={}, availability-zones={}, tags={}) ...",
+            instanceStates,
+            availabilityZones,
+            tags
+        );
```

Comment:
It's querying IMDS for the instances.

I've added this at `info` because logs don't tell you anything about what's going on and when the plugin doesn't work you have little visibility